pkg/cmd/miscellaneous.go

```diff
@@ -356,5 +342,9 @@ func getPublicIP() string {
 	defer resp.Body.Close()
 	ip, err := ioutil.ReadAll(resp.Body)
 	checkError(err)
+	if !isIP(string(ip)) {
```

Comment:
```suggestion
	if !isIPv4(string(ip)) {
```
shouldn't this be isIPv4?

Reply:
yes thanks.